SwiftCheck/Test.swift

```diff
@@ -6,6 +6,78 @@
 //  Copyright (c) 2015 TypeLift. All rights reserved.
 //
 
+// MARK: - Testing
+
+/// Property Testing is a more static and expressive form of Test-Driven Development that emphasizes
+/// the testability of program properties - A statement or invariant that can be proven to hold when
+/// fed any number of arguments of a particular kind.  It is akin to Fuzz Testing but is made
+/// significantly more power by the primitives in this framework.
+///
```

Comment:
At this point, I'm really interested. You've caught my attention, even if I've never heard of QuickCheck before. You're assuming your reader is familiar with TDD, but that's not a terrible assumption (and a motivated reader will have a good intuition for what you mean at this point anyway).